### PR TITLE
Feature/dependencies cutting

### PR DESCRIPTION
**Pull request by i-maslennikov**

### CODE REVIEW

File: Storage.EntityFramework/Converters/Domain2Persistent/IDomain2PersistentConverter.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace NuClear.Storage.EntityFramework.Converters.Domain2Persistent
+{
+    public interface IDomain2PersistentConverter<in TDomainEntity, out TPersistentEntity>
```
Comment (denisivan0v) [inline Storage.EntityFramework/Converters/Domain2Persistent/IDomain2PersistentConverter.cs]: The symbol "2" in public abstraction name is confusing a little.
Let's change the abstraction name to, say, IPersistenceObjectBuilder

File: Storage.EntityFramework.DI.Unity/EFDbModelMassProcessor.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        private readonly IUnityContainer _container;
+        private readonly List<Type> _configurations = new List<Type>();
+        private readonly List<Type> _conventions = new List<Type>();
+
+        public EFDbModelMassProcessor(IUnityContainer container)
+        {
+            _container = container;
+        }
+
+        public Type[] GetAssignableTypes()
```
Comment (denisivan0v) [inline Storage.EntityFramework.DI.Unity/EFDbModelMassProcessor.cs]: Noone calls GetAssignableTypes method

**Comment by i-maslennikov** [comment]
Changes can have effect on several issuses:
resolve #9
resolve #7
resolve #6

File: Storage.EntityFramework/Converters/Domain2Persistent/IDomainToPersistentObjectConverter.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿namespace NuClear.Storage.EntityFramework.Converters.Domain2Persistent
```
Comment (denisivan0v) [inline Storage.EntityFramework/Converters/Domain2Persistent/IDomainToPersistentObjectConverter.cs]: Let's update namespace NuClear.Storage.EntityFramework.Converters.Domain2Persistent too

**Pull request merged**